index.js

```diff
@@ -127,6 +131,10 @@ function session(options) {
     saveUninitializedSession = true;
   }
 
+  if (alwaysTouchUnmodified === undefined) {
+    alwaysTouchUnmodified = true;
```

Comment:
I forget looking through the comments: is there an intention to make `alwaysTouchUnmodifed` the default behavior in the next major? I ask because all default options that are Booleans should be `false` such that `undefined` and `null` are stand in values. The only options that default to `true` are deprecated options, but this doesn't have a deprecation warning on it.